.gitignore

```diff
@@ -10,3 +10,4 @@ __pycache__
 .mypy_cache
 .pytest_cache
 .python-version
+.venv
```

Comment:
Are you not using `pyenv`?

Reply:
@jsf9k no i'm using virtualenv using `python -m venv` I make sure to create / recreate the virtual environment using the python version using the latest stable version.

Reply:
I'll resolve this, but I highly suggest using `pyenv` over the `venv` module.  It's a game changer.